.github/workflows/integration-tests.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: denoland/setup-deno@v1
         if: '${{!steps.release-check.outputs.IS_RELEASE}}'
         with:
-          deno-version: v1.x
+          deno-version: v1.44.4
```

Comment:
Our supported deno version range is:
```
const DENO_VERSION_RANGE = '1.37.0 - 1.44.4'
```

Latest is 1.46.3 now, so CLI tries to download one instead of using one we install ourselves in GHA and that is very flaky in Windows tests, so instead let's install Deno version that matches supported range in first place